### ISSUE TITLE
Is OMEMO working if Conversations has OMEMO grey out and can't be toggled?

### ISSUE THREAD

**Issue by licaon-kter** [ReneVolution/profanity-omemo-plugin#30]
...while talking to profanity 0.51 + omemo-master

**Comment by ReneVolution** [comment]
No. Does not sound like it would. So the profanity client has not announced it's omemo capabilities.

**Comment by ReneVolution** [comment]
Did you install the `profanity-omemo-plugin` and turned it on in profanity?

**Comment by licaon-kter** [comment]
Yes it was on, and otherwise unencrypted worked ok though.

**Comment by ReneVolution** [comment]
Thanks for the feedback. Any chance you could send some logs?

**Comment by licaon-kter** [comment]
I'll have to retest (to weed out any user error too), and if it happens again I'll attach some logs.

**Comment by licaon-kter** [comment]
Ok so, I switch OMEMO on in Conversations and when trying to send a message I get "Error fetching OMEMO key!". Nothing logged in Profanity when Conversations tries to get the keys. But, the Android log says: https://gist.github.com/licaon-kter/0b24a614038eb32f759585f1f8bd1986 (ME is on Profanity, CONTACT is on Conversations)

Trying to send from Profanity, see [this comment](https://github.com/ReneVolution/profanity-omemo-plugin/issues/25#issuecomment-331678631)

**Comment by ReneVolution** [comment]
So. The issue comes from the profanity omemo plugin failing to announce it‘s bundle info due to a decoding issue when converting an XML to string. 

I have fixed a typo in the code now to help with that. Can you please retest with current git master?

**Comment by licaon-kter** [comment]
delete folder
delete omemo installed files
git clone 982612f 
install.sh
/connect
/sub allow
send text message, message arrives in Conversations, unencrypted, log says: https://gist.github.com/licaon-kter/c02b4be6766c3160503a9490feb7b978

In Conversations (both presence toggles ON), enable OMEMO, write a message, send: Error - No keys, logcat: https://gist.github.com/licaon-kter/735ffa30f1ef68766449111842616fc5

(ME is on Profanity, CONTACT is on Conversations)

**Comment by ReneVolution** [comment]
Thanks a lot for this. I need to setup a machine (docker) that uses Python3.6, maybe I can reproduce then. Unfortunately, using pyenv doesn't compile profanity yet. So I may need to take a different approach :) ... just you know I am on it.
